docs/testing.md

```diff
@@ -156,6 +156,35 @@ describe PostPolicy do
 end
 ```
 
+### Testing params scopes
```

Comment:
Let's make this a subsection of `Testing scopes`, smth like:

```suggestion
#### Action Controller params example
```

And add a similar heading for Active Record (`Active Record relation example`).